certified-connectors/E-Sign/apiDefinition.swagger.json

```diff
@@ -2,12 +2,12 @@
   "swagger": "2.0",
   "info": {
     "title": "E-Sign",
-    "description": "E-Sign Anything. Anywhere. It’s easy to e-sign and send your documents on any device with our secure electronic signature system.",
-    "version": "3.1",
+    "description": "E-Sign Anything. Anywhere. It's easy to e-sign and send your documents on any device with our secure electronic signature system.",
+    "version": "5.0",
     "contact": {
-      "name": "E-Sign Support",
-      "url": "https://www.e-sign.co.uk",
-      "email": "support@e-sign.co.uk"
+        "name": "E-Sign Support",
```

Comment:
@eSignHQ can you please fix the indentation here?

Reply:
@v-EgorKozhadei fixed now

Reply:
@eSignHQ  There's still a difference in one whitespace between the existing version and the updated one. Can you please fix it one more time?

Reply:
@v-EgorKozhadei ok this should be ok now